adapters/valueimpression/usersync_test.go

```diff
@@ -0,0 +1,25 @@
+package valueimpression
+
+import (
+	"testing"
+	"text/template"
+
+	"github.com/prebid/prebid-server/privacy"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValueImpressionSyncer(t *testing.T) {
+	syncURL := "https://rtb.valueimpression.com/usersync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirectUri=http%3A%2F%2Flocalhost:8000%2Fsetuid%3Fbidder%3Dvalueimpression%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26us_privacy%3D{{.USPrivacy}}%26uid%3D%24UID"
```

Comment:
Please omit `us_privacy` from the direct url. Prebid Server does not use that value during a callback. It will, instead,  never request a cookie sync if CCPA is enforced and opted out.